File: hw/nase/u3/hw24CompositePattern/CompossitePattern.python/model/clerk.py
# Jennyfer Nase 

from model.employee import Employee

class Clerk(Employee):
    def __init__(self, a_name=None):
        super().__init__()
        self.title = "Clerk"
        if a_name is not None:
            self.name = a_name

    def state_name(self):
        super().state_name()

File: hw/nase/u3/hw24CompositePattern/CompossitePattern.java/Employee/src/main/java/espe/edu/ec/employee/model/Clerk.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package espe.edu.ec.employee.model;

/**
 *
 * @author Jennyfer Nase
 */
public class Clerk extends Employee {

    public Clerk(String aName) {
        this();
        this.name = aName;
    }

    public Clerk() {
        this.title = "Clerk";
    }
    
    @Override
    public void stateName() {
        super.stateName();
    }
}